src/applications/mhv/medications/actions/prescriptions.js

```diff
@@ -2,8 +2,8 @@ import { Actions } from '../util/actionTypes';
 import { getPrescription, getPrescriptionList, fillRx } from '../api/rxApi';
 import { getAllergies } from '../../medical-records/api/MrApi';
 
-export const setSortEndpoint = sortEndpoint => async dispatch => {
-  dispatch({ type: Actions.Prescriptions.SET_SORT_ENDPOINT, sortEndpoint });
+export const setSortListOption = sortOption => async dispatch => {
+  dispatch({ type: Actions.Prescriptions.SET_SORT_OPTION, sortOption });
 };
```

Comment:
It would be way better to remove any Redux related logic for sorting selection, and just pass the value down to `<MedicationsList />` from `<Prescriptions />`. I am currently working on my PR which would clean the Redux logic  for sorting selection and retain its via `sessionStorage`. I suggest to please wait for my PR first. Thanks!